### DISCUSSION TITLE
Support for servers on earlier OS's?

### DISCUSSION THREAD

**Discussion by PythonBlue** [apohl79/audiogridder#985] [Ideas]
Let me start off by saying that I did check for related issues. #681 is along the lines of what I'm driving at, but it's not identical, either.

It seems to me like this project could really shine if it was a way to get older VST's to cooperate on modern platforms. For example, I found this very useful for getting the old Emulator/Proteus X VST's and sound libraries to work in my Logic Pro setup on my Mac (especially significant because said VST's were Windows exclusive), though even there, all that works without use of jBridge is the 64-bit build of Emulator X3 due to the 64-bit requirement of the current builds of AudioGridder.

I realize this would be quite a task to make it more compatible, in light of the response given in the issue I linked earlier, but it's also important to note: the NetVST project, while more limited, does support servers on older 32-bit Windows OS's.

Thank you for your time, and be assured that I am impressed with this idea all the same!

**Comment by apohl79** [comment]
I see your point. I will create a feature request issue for now. There might be someone looking at that sometime. Cheers.

**Comment by nesgovoroff** [comment]
+1 for older OS's. 
it would be fantastic to use plugins and gear such as Focusrite LiquidMix or TC Powercore running on older mac and transferred to working modern computer.

**Reply by nesgovoroff** [reply]
BTW, I've tried installing AudioGridder on MacOS 10.8, it installed, but failed to scan 32-bit plugins. LiquidMix fails to scan.
